docs/pages/index.md

```diff
@@ -63,19 +63,19 @@ PM> Install-Package BlazorWebView.Wpf -IncludePrerelease
 ### Add BlazorWebView to your Activity/View, ViewController/View, or Window
 
 Next add the BlazorWebView (it's named like this in every package) to your Activity / View (for Android),
-ViewController / View (for iOS and Mac), or to your window Xaml. Make sure that the BlazorWebView gets an
+ViewController / View (for iOS and Mac), or to your window XAML. Make sure that the BlazorWebView gets an
 identifier so we can reference it in a code behind file.
 
-### Wire up your blazor project to the BlazorWebView
+### Wire up your Blazor project to the BlazorWebView
 
-First, we need to adapt the url to the blazor javascript. It will be loaded from the nuget assembly by
-referencing a dedicated scheme. The Uri to the blazor javascript is:
+First, we need to adapt the URL to the Blazor JavaScript. It will be loaded from the NuGet assembly by
+referencing a dedicated scheme. The URL to the Blazor JavaScript is:
```

Comment:
This one originally said `Uri` (rather than `Url`), but I could see no reason for that. For consistency's sake, using `URI` in all places.